app/pages/reset-password/new-password-form.jsx

```diff
@@ -46,29 +54,29 @@ const NewPasswordForm = ({ onSubmit, disabled, inProgress, resetSuccess, resetEr
         <i className="fa fa-check-circle form-help success" />}
 
       {resetError &&
-        <Translate
-          className="form-help error"
-          component="small"
-          content="resetPassword.resetError"
-        />}
+        <small className="form-help error">
```

Comment:
Why has `Translate` been removed?

Reply:
Because the resetError text is an error response from the API now and will vary based on the api error.